deployments/container/Makefile

```diff
@@ -92,14 +84,8 @@ $(IMAGE_TARGETS): image-%:
 		-f $(DOCKERFILE) \
 		$(CURDIR)
 
-# Handle the default build target.
-.PHONY: build
-build: $(DEFAULT_PUSH_TARGET)
-$(DEFAULT_PUSH_TARGET): build-$(DEFAULT_PUSH_TARGET)
-$(DEFAULT_PUSH_TARGET): DIST = $(DEFAULT_PUSH_TARGET)
-
-REGCTL ?= regctl
-$(PUSH_TARGETS): push-%:
+# Push the image to the registry
+push:
 	$(REGCTL) \
```

Comment:
In our day to day, we don't really run `regctl` locally.

In GHA CI, we run `make -f deployments/container/Makefile build` and have `buildx` push-on-build (also not involving `regctl`). Do we want / need this change on this PR?

Reply:
I am not introducing a `regctl` change here.  I just moved the VAR declaration up. but no behaviour/functionality modification

Reply:
> I just moved the VAR declaration up.

gotcha!